src/styles/components/_pagination.scss

```diff
@@ -0,0 +1,49 @@
+.pagination {
+  background: white;
+  border: 0.1rem solid rgba(0, 0, 0, 0.15);
+  width: fit-content;
+  margin: 0 auto;
+}
+
+.pagination a {
+  padding: 0.8rem 1.5rem 0.8rem 1.5rem;
+  display: table-cell;
+  text-decoration: none;
+  overflow: hidden;
+}
+
+.pagination a:hover {
+  background: #f2f2f2;
+  text-align: center;
+}
+
+.pagination .next-page {
+  height: 0;
+  width: 0;
+  margin-top: 0.2rem;
+  border-left: 1.7rem solid #0b41cd;
+  border-bottom: 1rem solid transparent;
+  border-top: 1rem solid transparent;
+}
+
+.pagination .prev-page {
+  height: 0;
+  width: 0;
+  margin-top: 0.2rem;
+  border-right: 1.7rem solid #0b41cd;
+  border-bottom: 1rem solid transparent;
+  border-top: 1rem solid transparent;
+}
+
+.prev-page.caret-left-arrow-disabled {
+  border-right-color: #d8d8d8;
+}
+
+.next-page.caret-right-arrow-disabled {
+  border-left-color: #d8d8d8;
+}
```

Comment:
This has the same styling with `.prev-page.caret-left-arrow-disabled`. Consider merging them.

Reply:
One is ` border-left-color`, the other is `border-right-color`.